src/Symfony/Component/Validator/Mapping/Cache/Psr6Cache.php

```diff
@@ -66,6 +66,6 @@ public function write(ClassMetadata $metadata)
      */
     private function escapeClassName(string $class): string
     {
-        return str_replace('\\', '.', $class);
+        return rawurlencode($class);
```

Comment:
ok, so last comment: let's remove the escapeClassName method altogether, it's only overhead :)